plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceBeanV4.java

```diff
@@ -0,0 +1,314 @@
+
+    private Instant date;
+    private String account;
+    private String userId;
+    private String operation;
+    private Currency coin;
+    private BigDecimal change;
+    private String remark;
+
+    private int rowId;
+    public List<Integer> usedIds = new ArrayList<>();
+
+    private boolean isInTransaction;
+    private boolean unsupportedRow;
+    private String message;
+    private boolean isMergedWithAnotherGroup;
+
+    private Currency marketBase;
+    private Currency marketQuote;
+    private TransactionType type;
+    private BigDecimal amountBase;
+    private BigDecimal amountQuote;
+    private BigDecimal fee;
+    private Currency feeCurrency;
+    private BigDecimal transactionPrice;
+
+    public List<BinanceBeanV4> feeTransactions = new ArrayList<>();
+
+    @Parsed(field = "User_ID")
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    @Parsed(field = "UTC_Time")
+    @Format(formats = {"yyyy-MM-dd HH:mm:ss"}, options = {"locale=EN", "timezone=UTC"})
+    public void setDate(@NonNull Date value) {
+        date = value.toInstant();
+    }
+
+    public void setDate(Instant value) {
+        date = value;
+    }
+
+    @Parsed(field = "Account")
+    public void setAccount(String account) {
+        var supportedAccounts = BinanceSupportedOperations.supportedAccountTypes;
+        if (!supportedAccounts.contains(account)) {
+            this.setUnsupportedRow(true);
+            this.setMessage("Unsupported type of account " + account + " ");
+        }
+        this.account = account;
+    }
+
+    @Parsed(field = "Operation")
+    public void setOperation(String operation) {
+        var supportedOperations = BinanceSupportedOperations.supportedOperationTypes;
+        if (!supportedOperations.contains(operation)) {
+            this.setUnsupportedRow(true);
+            this.setMessage("Unsupported type of operation " + operation + " ");
+        }
+        this.operation = operation;
+        this.type = BinanceSwitcher.operationTypeSwitcher(operation);
+    }
+
+    @Parsed(field = "Coin")
+    public void setCoin(String coin) {
+        try {
+            this.coin = Currency.fromCode(coin);
+        } catch (IllegalArgumentException e) {
+            this.setUnsupportedRow(true);
+            this.setMessage(e.getMessage());
+        }
+    }
+
+    public void setCoin(Currency coin) {
+        this.coin = coin;
+    }
+
+    @Parsed(field = "Change")
+    public void setChange(String change) {
+        try {
+            this.change = new BigDecimal(change);
+        } catch (NumberFormatException e) {
+            this.setUnsupportedRow(true);
+            this.setMessage(e.getMessage());
+        }
+    }
+
+    public void setChange(BigDecimal change) {
+        this.change = change;
+    }
+
+    @Parsed(field = "Remark")
+    public void setRemark(String remark) {
+        this.remark = remark;
+    }
+
+    public Currency getCoin() {
```

Comment:
Dont specify simple getters and setters manually. Use lomboks `@Data`  or another annotations.   See `@Data` javadoc for more info. Here specify only getters and setters you want to somehow modify (for example add annotation)